### DISCUSSION TITLE
can wiki.js v2 use one local  git repository instead of one git server to back up?

### DISCUSSION THREAD

**Discussion by zhuzhzh** [requarks/wiki#5281] [General]
I create one directory  /usr/wiki.
Then in /usr/wiki,  I input `git init`.

in wiki.js -> storage -> git,  can I config it as the following?

ssh
ssh://127.0.0.1/usr/wiki
path
~/.ssh

I will generte the key with ssh-keygen in ~/.ssh and put `local host` into know_hosts

I did this, but it does't work.

![图片](https://user-images.githubusercontent.com/14127098/167818292-39f5650b-b326-4d2e-a537-a5e49c69f25b.png)

**Comment by jarrodlovas** [comment]
Yeah, It should be able to use your local git server.
I've found if your wiki instance is on a docker container that the ssh keys need to be inside the container for the auth to work otherwise it kept erroring for me. That's how I got mine to work with ssh. I also switch the ssh -private key mode to contents since I could not get path to work
